homepage.html

```diff
@@ -13,7 +13,7 @@
     />
     <meta name="author" content="LayoutIt!" />
 
-    <!-- <link href="css/bootstrap.min.css" rel="stylesheet" /> -->
+      <!-- <link href="css/bootstrap.min.css" rel="stylesheet" />   -->
```

Comment:
why add a space here?

Reply:
I was trying to use this link and I failed. There is a conflict. I will fix it soon.